api/test/wfl/integration/xx_pipeline_test.clj

```diff
@@ -0,0 +1,120 @@
+(ns wfl.integration.xx-pipeline-test
+  (:require [clojure.test :refer [deftest testing is] :as clj-test]
+            [wfl.service.cromwell :refer [wait-for-workflow-complete]]
+            [wfl.tools.endpoints :as endpoints]
+            [wfl.tools.fixtures :as fixtures]
+            [wfl.tools.workloads :as workloads]
+            [wfl.module.xx :as xx]
+            [wfl.service.postgres :as postgres]
+            [wfl.jdbc :as jdbc]
+            [wfl.util :as util]
+            [clojure.string :as str])
+  (:import (java.util UUID)
+           (java.time OffsetDateTime)))
+
+(clj-test/use-fixtures :once fixtures/clean-db-fixture)
+
+(defmacro is-not
+  ([form] `(is-not ~form nil))
+  ([form msg] `(is (not ~form) ~msg)))
```

Comment:
I assume `is-not` doesn't confuse the test parser.

Reply:
It didn't. As you suggested in your slack comment, I removed it.